src/renderers/WebGLRenderer.js

```diff
@@ -1437,7 +1437,7 @@ function WebGLRenderer( parameters = {} ) {
 
 		const fog = scene.fog;
 		const environment = material.isMeshStandardMaterial ? scene.environment : null;
-		const encoding = ( _currentRenderTarget === null ) ? _this.outputEncoding : ( _currentRenderTarget.isXRRenderTarget === true ? _currentRenderTarget.texture.encoding : LinearEncoding );
+		const encoding = ( _currentRenderTarget === null ) ? _this.outputEncoding : _currentRenderTarget.texture.encoding;
```

Comment:
I'm afraid this change is not right. If a render target is of type `SRGB8_ALPHA8`, WebGL automatically holds framebuffer data as sRGB. 

This line only controls the use case when a _manual_ sRGB encode is required. That is only necessary when rendering to screen OR when using a render target in context of XR.